test/SpokePool.Relay.ts

```diff
@@ -89,6 +105,40 @@ describe("SpokePool Relayer Logic", async function () {
     // Fill amount should be equal to full relay amount.
     expect(await spokePool.relayFills(relayHash)).to.equal(fullRelayAmount);
   });
+  it("Repayment chain is set correctly", async function () {
+    // Can set repayment chain if full fill.
+    const { relayData } = getRelayHash(
+      depositor.address,
+      recipient.address,
+      consts.firstDepositId,
+      consts.originChainId,
+      consts.destinationChainId,
+      destErc20.address
+    );
+
+    // Changed consts.amountToRelay to relayData.amount to make it a full fill
+    await expect(
+      spokePool.connect(relayer).fillRelay(...getFillRelayParams(relayData, relayData.amount, consts.repaymentChainId))
+    )
+      .to.emit(spokePool, "FilledRelay")
+      .withArgs(
+        relayData.amount,
+        relayData.amount,
+        relayData.amount,
+        consts.repaymentChainId,
```

Comment:
Should you check in this test that `fillCounter` is not incremented because its not a `localRepayment`?

Reply:
I've done that^